tests/jballerina-unit-test/src/test/resources/test-src/expressions/access/member_access_negative.bal

```diff
@@ -196,9 +196,16 @@ function testInvalidRecordMemberAccessByConst() {
     int i2 = e[AGEC];
 }
 
+type Abc record {
+    string[] s;
+    float[] v;
+};
+
 function testMemberAccessWithoutIndex() {
     int[] intArray = [];
-    int value = intArray[];
+    Abc a = {s: ["1.0", "2.0", "3.0"], v: [1.0, 2.0, 3.0]};
+    a.s[] = "4.1";
+    a.v[1] = intArray[];
     intArray[]
 }
```

Comment:
Shall we write a test case for the exact two cases mentioned in the issue?

Reply:
Okay. I'll add a test.